services/storage/docker/boot.sh

```diff
@@ -42,10 +42,32 @@ APP_LOG_LEVEL=${STORAGE_LOGLEVEL:-${LOG_LEVEL:-${LOGLEVEL:-INFO}}}
 SERVER_LOG_LEVEL=$(echo "${APP_LOG_LEVEL}" | tr '[:upper:]' '[:lower:]')
 echo "$INFO" "Log-level app/server: $APP_LOG_LEVEL/$SERVER_LOG_LEVEL"
 
-if [ "${SC_BOOT_MODE}" = "debug" ]; then
-  reload_dir_packages=$(find /devel/packages -maxdepth 3 -type d -path "*/src/*" ! -path "*.*" -exec echo '--reload-dir {} \' \;)
+if [ "${STORAGE_WORKER_MODE}" = "true" ]; then
+  if [ "${SC_BOOT_MODE}" = "debug" ]; then
+    exec watchmedo auto-restart \
+      --directory /devel/packages \
+      --directory services/storage \
+      --pattern "*.py" \
+      --recursive \
+      -- \
```

Comment:
is this correct. looks like a flag is missing `-- \`

Reply:
no that is the way.